### DISCUSSION TITLE
Fingerprint Scanner - TTL (GT-521F32) Supported ?

### DISCUSSION THREAD

**Discussion by Noschvie** [arendst/Tasmota#11815] [Q&A] [answered]
Is the Fingerprint Scanner GT-521F32 supported by Tasmota ?
Thanks!

**Comment by sfromis** [comment, marked as answer]
Unless compatible with AS608, the answer appears to be "no", unless someone creates a driver, of course.
https://tasmota.github.io/docs/Supported-Peripherals/

**Reply by Noschvie** [reply]
The driver to be created have to be based on this library ?
https://github.com/PawelMiera/sparkfun-Fingerprint_Scanner-TTL-for-esp32
